app/src/main/java/org/mozilla/tv/firefox/webrender/EngineViewLifecycleFragment.kt

```diff
@@ -34,44 +34,44 @@ abstract class EngineViewLifecycleFragment : LocaleAwareFragment() {
      * The [EngineView] in use by this fragment. If the value is non-null, the EngineView is present
      * in the view hierarchy, null otherwise.
      */
-    var webView: EngineView? = null
+    var engineView: EngineView? = null
         @UiThread get // On a background thread, it may have been removed from the view hierarchy.
         private set
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        webView = (view.findViewById<View>(R.id.webview) as EngineView).apply {
-            onWebViewCreated(this)
+        engineView = (view.findViewById<View>(R.id.engineView) as EngineView).apply {
+            onEngineViewCreated(this)
         }
     }
 
-    open fun onWebViewCreated(webView: EngineView) = Unit
+    open fun onEngineViewCreated(engineView: EngineView) = Unit
 
     override fun onStop() {
         super.onStop()
 
         // NB: onStop unexpectedly calls onPause: see below.
         //
         // When the user says "Alexa pause [the video]", the Activity will be paused/resumed while
-        // Alexa handles the request. If the WebView is paused during video playback, the video will
+        // Alexa handles the request. If the EngineView is paused during video playback, the video will
```

Comment:
Do we have a concept of EngineView being paused? I could see this being "webView inside EngineView", but just wanted to clarify.

Reply:
We do have `EngineView.onPause()`